src/site/twitch.tv/modules/chat/ChatList.vue

```diff
@@ -198,6 +199,11 @@ function onChatMessage(msg: ChatMessage, msgData: Twitch.AnyMessage, shouldRende
 			msg.setHighlight("#3296e6", "Returning Chatter");
 		}
 
+		// Assign highlight to your own message
+		if (msgData.nonce && showYouHighlights.value) {
```

Comment:
`nonce` is undefined if the message isn't sent directly from the current tab. I believe it may be best to make a author ID check.